chronograf.go

```diff
@@ -736,22 +737,17 @@ type OrganizationsStore interface {
 	DefaultOrganization(ctx context.Context) (*Organization, error)
 }
 
-// AuthConfig is the global application config section for auth parameters
-type AuthConfig struct {
-	// SuperAdminNewUsers should be true by default to give a seamless upgrade to
-	// 1.4.0 for legacy users. It means that all new users will by default receive
-	// SuperAdmin status. If a SuperAdmin wants to change this behavior, they
-	// can toggle it off via the Chronograf UI, in which case newly authenticating
-	// users will simply receive whatever role they would otherwise receive.
-	SuperAdminNewUsers bool `json:"superAdminNewUsers"`
-}
-
 // Config is the global application Config for parameters that can be set via
 // API, with different sections, such as Auth
 type Config struct {
 	Auth AuthConfig `json:"auth"`
 }
 
+// AuthConfig is the global application config section for auth parameters
+type AuthConfig struct {
+	SuperAdminNewUsers bool `json:"superAdminNewUsers"`
```

Comment:
What happened to the comments that used to be here?

Reply:
the comment that was there was obsolete and incorrect. we'll add a new one to explain its purpose